modules/sdk-coin-apt/src/lib/utils.ts

```diff
@@ -96,6 +100,18 @@ export class Utils implements BaseUtils {
     return deserializer.deserialize(SignedTransaction);
   }
 
+  deserializeVectorOfAddresses(addressBuffer: Uint8Array): AccountAddress[] {
+    const deserializer = new Deserializer(addressBuffer);
+    const deserializedAddresses = deserializer.deserializeVector(AccountAddress);
+    return deserializedAddresses;
+  }
+
+  deserializeVectorOfAmounts(amountBuffer: Uint8Array): U64[] {
```

Comment:
using generic names is better for utility functions, for reusability.
```suggestion
  deserializeU64Vector(serializedBytes: Uint8Array): U64[] {
```

Reply:
changed.

Reply:
pls modify the param name also as suggested above.